nuget.config

```diff
@@ -2,9 +2,20 @@
 <configuration>
   <config>
     <add key="repositorypath" value="packages" />
+    <add key="signatureValidationMode" value="require" />
   </config>
   <packageSources>
+    <clear />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
-    <add key="vs-devcore" value="https://www.myget.org/F/vs-devcore/api/v3/index.json" />
+    <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
```

Comment:
I wonder if we could remove this package source? Could we use only nuget?

Reply:
we have to use vs-impl for microbuild components